Jenkinsfile

```diff
@@ -0,0 +1,151 @@
+AGENT_YAML = '''
+    apiVersion: v1
+    kind: Pod
+    spec:
+      containers:
+      - name: maven
+        image: maven:3.8.6-openjdk-8-slim
+        tty: true
+        resources:
+          requests:
+            memory: "1Gi"
+            cpu: "500m"
+          limits:
+            memory: "1Gi"
+            cpu: "500m"
+        command:
+        - cat
+'''
+
+pipeline {
+
+    agent {
+        kubernetes {
+            defaultContainer 'maven'
+            yaml AGENT_YAML
+        }
+    }
+
+    environment {
+        GPG_SECRET     = credentials('presto-release-gpg-secret')
+        GPG_TRUST      = credentials("presto-release-gpg-trust")
+        GPG_PASSPHRASE = credentials("presto-release-gpg-passphrase")
+
+        GITHUB_OSS_TOKEN_ID = 'github-personal-token-wanglinsong'
```

Comment:
Use a bot account please

Reply:
I will rename this, but the underline credential is still from my account, unless you share another github API token with me.

Reply:
updated.